### DISCUSSION TITLE
Duda acerca de referencias en informe

### DISCUSSION THREAD

**Discussion by BenjaCarrill0** [IIC2133-PUC/2024-1#46] [Tarea 0] [answered]
Hola! Una consulta, si utilizo un algoritmo visto en clases en la tarea, ¿es necesario el referenciarlo? de ser así ¿cómo se debería referenciar?

**Comment by benjavicente** [comment, marked as answer]
Hola! Toda materia vista en clases se asume, no hay que referenciarla.

Solo se referencia recursos fuera de lo que se ve en clases.

**Reply by BenjaCarrill0** [reply]
Graciass
